locations/spiders/gls-deu.py

```diff
@@ -0,0 +1,57 @@
+import scrapy
+import json
+import csv
+from locations.items import GeojsonPointItem
+
+HEADERS = {"X-Requested-With": "XMLHttpRequest"}
+STORELOCATOR = "https://api.gls-pakete.de/parcelshops?version=4&coordinates={},{}&distance=40"
+
+class GenspiderSpider(scrapy.Spider):
```

Comment:
The class name should be related to the spider. Can you change this to `GLSSpider` or `GeneralLogisticsSystemsSpider`?